src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/4.sql

```diff
@@ -2154,41 +2157,27 @@ END
 GO
 
 /*************************************************************
-    Search Parameter Status Registry
+    Search Parameter Status Information
 **************************************************************/
 
-CREATE TYPE dbo.SearchParamStatusRegistryTableType_1 AS TABLE
+CREATE TYPE dbo.SearchParamTableType_1 AS TABLE
```

Comment:
What is the _1 suffix for?

Reply:
I admit I'm not super sure. Other `CREATE TYPE` statements for table valued parameters are followed by a `_1` - that being said, I can't seem to find anything about that in the [style guide](https://github.com/ktaranov/sqlserver-kit/blob/master/SQL%20Server%20Name%20Convention%20and%20T-SQL%20Programming%20Style.md).

Reply:
Oh I remember now. It's because the table types are immutable and can't be altered.